internal/plugin_config_test.go

```diff
@@ -1025,6 +1025,12 @@ func TestConfigInvalidEvalInterval(t *testing.T) {
 			`the policy policy-app has an invalid policy.evaluationInterval.noncompliant value: time: unknown unit ` +
 				`"w" in duration "1w2d"`,
 		},
+		{
+			`{"compliant": "watch", "noncompliant": "watch"}`,
+			`{"compliant": "watch", "noncompliant": "watch"}`,
+			`{"compliant": "watch", "noncompliant": "watch"}`,
+			``,
+		},
```

Comment:
With this here, this is now testing the evaluation interval in general. Would you rename the test to `TestConfigEvalInterval`?